lib/web_of_science/base_rest_retriever.rb

```diff
@@ -18,23 +18,15 @@ def initialize(path, params, batch_size: MAX_RECORDS)
       @batch_size = batch_size
     end
 
-    # Retrieve and collect all record UIDs
-    # @return [Array<String>] WosUIDs
-    def merged_uids
-      uids = batch_one.uids
-      uids += next_batch.uids while next_batch?
-      uids
-    end
-
```

Comment:
this inefficient method is going away, to be supplanted by our new more efficient method in `UidsQueryRestsRetriever`

the only consumer of it will now use our new method